includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -304,6 +303,12 @@ public function wc_setup_store_setup() {
 			$state = '*';
 		}
 
+		$locale_info = include( WC()->plugin_path() . '/i18n/locale-info.php' );
+		function get_currency_code( $country ) {
+			return $country[ 'currency_code' ];
```

Comment:
Minor linting error - there shouldn't be a space for the array key if it's a string. Should be `$country['currency_code']`.